pkg/kn/commands/configuration_edit_flags.go

```diff
@@ -18,21 +18,40 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/api/resource"
+
 	servinglib "github.com/knative/client/pkg/serving"
 	servingv1alpha1 "github.com/knative/serving/pkg/apis/serving/v1alpha1"
-	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
 )
 
 type ConfigurationEditFlags struct {
-	Image string
-	Env   []string
+	Image         string
+	Env           []string
+	RequestsFlags RequestsFlags
+	LimitsFlags   LimitsFlags
+}
+
+type RequestsFlags struct {
+	CPU    string
+	Memory string
+}
+
+type LimitsFlags struct {
+	CPU    string
+	Memory string
 }
 
 func (p *ConfigurationEditFlags) AddFlags(command *cobra.Command) {
 	command.Flags().StringVar(&p.Image, "image", "", "Image to run.")
 	command.Flags().StringArrayVarP(&p.Env, "env", "e", []string{},
 		"Environment variable to set. NAME=value; you may provide this flag "+
 			"any number of times to set multiple environment variables.")
+	command.Flags().StringVar(&p.RequestsFlags.CPU, "requests-cpu", "", "The requested CPU (e.g., 250m).")
+	command.Flags().StringVar(&p.RequestsFlags.Memory, "requests-memory", "", "The requested CPU (e.g., 64Mi).")
+	command.Flags().StringVar(&p.LimitsFlags.CPU, "limits-cpu", "", "The limits on the requested CPU (e.g., 1000m).")
+	command.Flags().StringVar(&p.LimitsFlags.Memory, "limits-memory", "", "The limits on the requested CPU (e.g., 1024Mi).")
 	command.MarkFlagRequired("image")
```

Comment:
:+1:  good seeing support for that here 

Ignore my q, but what about live/ready probes ?

Reply:
"live/ready probes?" can you elaborate? Drawing a blank...

Reply:
> My question: Do we really need to verify that `vendor/modules.txt` is always created the same bit-by-bit ? I mean the content is the same, just in a different order. If there is no order implied on modules.txt by `go mod vendor` I suggest adding a `sort` command into `hack/verify-codegen.sh` before diffing.

I would be cool with that? Can you include this in your PR or shall I?

Reply:
Update: I pushed a change to go.mod that I guess might have been needed. Let's see if that fixes the CI tests.

...the horrors of dealing with two dev machines that are not the same... :(